custom-words.txt

```diff
@@ -1122,6 +1123,7 @@ microservices
 microsofttranslator
 migratemysql
 migratetoaad
+Miliseconds
```

Comment:
typo - should be milliseconds

Reply:
Both of the entries in the custom words are misspellings of properties in our backend.